app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/AlertUserFragment.java

```diff
@@ -162,18 +174,6 @@ protected void onAddAllergens() {
 
     }
 
-
-    public static Integer getKey(HashMap<Integer, String> map, String value) {
```

Comment:
I couldn't see the alternate code for this method.

Reply:
He replace it o  line 51

Reply:
my bad thanks @ManognaMutyala